pallets/mips/src/lib.rs

```diff
@@ -465,6 +475,57 @@ impl<T: Trait> Module<T> {
             .collect()
     }
 
+    /// Retrieve votes for a proposal represented by MipsIndex `index`.
+    pub fn get_votes(index: MipsIndex) -> VoteCount<BalanceOf<T>>
```

Comment:
I suggest having different `impl` block for getters. Part of the best practice to segregate the state transition function/used internally and external getters.